packages/react-components/react-tabster/src/hooks/useModalAttributes.ts

```diff
@@ -5,10 +5,19 @@ import { useTabster } from './useTabster';
 
 export interface UseModalAttributesOptions {
   /**
-   * Traps focus inside the elements the attributes are applied
+   * Traps focus inside the elements the attributes are applied.
+   * Prefer this to `legacyTrapFocus`
+   * it forbids users to tab out of the focus trap into the actual browser.
    */
   trapFocus?: boolean;
 
+  /**
+   * Traps focus inside the elements the attributes are applied.
+   * It is recommended not to use the `absolute` variant since
+   * it forbigs users to tab out of the focus trap into the actual browser.
```

Comment:
```suggestion
   * Traps focus inside the elements the attributes are applied.
   * This prop enables legacy behavior to match previous versions of Fluent and is not
   * recommended for general use.
   * Enabling `legacyTrapFocus` prevents users from tabbing out of the focus trap and into
   * the actual browser. Prefer using `trapFocus` instead of this prop.
```